core/domain/feedback_domain_test.py

```diff
@@ -63,8 +63,8 @@ def test_get_last_two_message_ids_from_thread_with_many_messages(self):
         thread = feedback_domain.FeedbackThread(
             self.THREAD_ID, feconf.ENTITY_TYPE_EXPLORATION, self.EXP_ID,
             u'a_state_name', self.viewer_id, u'open', u'a subject', None, False,
-            5, # This value decides the number of messages.
-            fake_date, fake_date, 'last message', self.VIEWER_USERNAME)
+            # This value of "5" decides the number of messages.
```

Comment:
This and similar change are done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.